application/hubspot/v0/config/tasks.json

```diff
@@ -0,0 +1,1149 @@
+{
+  "$defs": {
+    "common": {
+      "contact-id-or-email": {
+        "instillAcceptFormats": ["string"],
+        "instillUIMultiline": true,
+        "instillUIOrder": 0,
+        "instillUpstreamTypes": ["value", "reference", "template"],
+        "title": "Contact ID or Email",
+        "type": "string"
+      },
+      "owner-id": {
+        "description": "The user who is assigned to the object",
+        "title": "Owner ID",
+        "instillUIOrder": 1,
+        "type": "string"
+      },
+      "associated-contact-id": {
+          "description": "Contact IDs associated with the object",
+          "title": "Associated Contact IDs",
+          "type": "array",
+          "instillFormat": "array:string",
+          "items": {
+            "title": "Associated Contact ID",
+            "type": "string",
+            "description": "Contact ID associated with the object",
+            "instillFormat": "string"
+          }
+      }
+    },
+    "contact": {
+      "email": {
+        "description": "Email address",
+        "instillUIOrder": 2,
+        "title": "Email Address",
+        "type": "string"
+      },
+      "first-name": {
+        "description": "First name",
+        "instillUIOrder": 3,
+        "title": "First Name",
+        "type": "string"
+      },
+      "last-name": {
+        "description": "Last name",
+        "instillUIOrder": 4,
+        "title": "Last Name",
+        "type": "string"
+      },
+      "phone-number": {
+        "description": "Phone number. Preferably, format is in country code + phone number. Example: +886xxxxxxxxx ",
+        "instillUIOrder": 5,
+        "title": "Phone Number",
+        "type": "string"
+      },
+      "company": {
+        "description": "Company",
+        "instillUIOrder": 6,
+        "title": "Company",
+        "type": "string"
+      },
+      "job-title": {
+        "description": "Job title",
+        "instillUIOrder": 7,
+        "title": "Job Title",
+        "type": "string"
+      },
+      "lifecycle-stage": {
+        "description": "Lifecycle stage. Default format is in small letters, all words are combined. Example: salesqualifiedlead. However, remember to check internal value for custom fields.",
+        "instillUIOrder": 8,
+        "title": "Lifecycle Stage",
+        "type": "string"
+      },
+      "lead-status": {
```

Comment:
Lead status is another property that user can select using drop down menu.
The official description for it: The contact's sales, prospecting or outreach status
![image](https://github.com/user-attachments/assets/994c649a-8d3f-4c63-8a52-9063892682e1)

I don't think it is possible to make it an enum because the values are customizable (users can add more values if they want).
![image](https://github.com/user-attachments/assets/98f2b23f-ffce-4fe3-80f0-9b301cb4c2ec)